packages/website/tests/index.spec.ts

```diff
@@ -1,7 +1,20 @@
 import AxeBuilder from '@axe-core/playwright';
-import { test } from '@playwright/test';
+import { expect, test } from '@playwright/test';
 
-test('Index', async ({ page }) => {
-  await page.goto('/');
-  await new AxeBuilder({ page }).analyze();
+test.describe('Website', () => {
+  test('Axe', async ({ page }) => {
+    await page.goto('/');
+    await new AxeBuilder({ page }).analyze();
+  });
+
+  test('should have no errors', async ({ page }) => {
+    const errorMessages = [] as string[];
+    page.on('console', msg =>
+      ['error', 'warning'].includes(msg.type())
+        ? errorMessages.push(`[${msg.type()}] ${msg.text()}`)
+        : null,
+    );
```

Comment:
Just nitpicking style here, we generally try to avoid using ternary conditions this way. It's kind of confusing to read (they normally are used to create some result value).

```suggestion
    page.on('console', msg => {
      if (['error', 'warning'].includes(msg.type())) {
        errorMessages.push(`[${msg.type()}] ${msg.text()}`);
      }
    });
```